Assets/HoloToolkit-Examples/SpectatorView/Scripts/ColorChanger.cs

```diff
@@ -56,7 +56,15 @@ private void Update()
         private void ChangeColor()
         {
             color = UnityEngine.Random.ColorHSV();
-            objectRenderer.material.color = color;
+            material.color = color;
+        }
+
+        /// <summary>
+        /// Destroys the changed material
```

Comment:
No mention of death here? 😋 https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1884/files/ff2249eb5d04e5bc85bcd8eed212fa68db602003..cded0c33ed890126d3dc6478403305a07e95067c#r185672508